pokedex/src/Components/Header/Header.jsx

```diff
@@ -0,0 +1,76 @@
+import logo from "../../assets/pokemonlogo.svg";
+import pokeball from "../../assets/pokeball.png";
+import { Link } from "react-router-dom";
+import "./Header.css";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+function Header({
+  pokemonList,
+  filteredPokemonList,
+  searchTerm,
+  setSearchTerm,
+}) {
+  const [selectedPokemon, setSelectedPokemon] = useState(null);
+  const navigate = useNavigate();
+
+  const showPokemon = async (url) => {
+    const response = await fetch(url);
+    if (!response.ok) {
+      console.error(`Error fetching Pokemon: ${response.statusText}`);
+      return;
+    }
+
+    const data = await response.json();
```

Comment:
Please, define `api` functions in `api.js` to separate the components from the api function defining